### DISCUSSION TITLE
Can not install the apk on samsung s7 tablet

### DISCUSSION THREAD

**Discussion by atilarey** [skyline-emu/skyline#1707] [General]
HI 
I've tried many times to install diferent versions oh the apk fli on my Samsung tab s7.
But always with the same result "the aplicattion has not been installed by a conflict with the packet"
I donˋt have any other versions installed before and I have persmissions with this apk.
Please, any idea how to resolve this issue?
Thank you so much

**Comment by mateoalfaro** [comment]
You need to find the APK you used before and you need to install it and uninstall it again,if that doesn't work go to settings apps and check on system apps and search for skyline if it's there then delete the app,and if that still doesn't work go to the discord to ask for help

**Reply by JeyRizo01** [reply]
The same thing happens to me, I have a one plus 7T Snapdragon 855+ processor and I can't install it and before I could do it

**Comment by atilarey** [comment]
Thank you so much for your answer!
But I haven't installed any previous version of the app, I've had always the same error when I tried to install it.
I've looked for any reference in the sistem's data folder, but it doesn' appears any trace abou it.
Maybe Could be a problem with this hardware?
Thank you again for your hep!

**Comment by atilarey** [comment]
anyone else with any problem as mine?
Thanks

**Reply by Crytonics** [reply]
hmm check in Android/data/ and see if there is folder skyline.emu.
If there is and apk is not installed then remove folder and try again install.

**Reply by atilarey** [reply]
Thank you but I've already checked data's folder and other trace about skyline, but there's nothing. I've looked for with different file browser app, and hidden folders
I don't know how install this app. I've tried everything...I'm a bit sad...
